frontend/src/scenes/billing/BillingCTAHero.tsx

```diff
@@ -0,0 +1,64 @@
+import { LemonButton } from '@posthog/lemon-ui'
+import { useActions, useValues } from 'kea'
+import { BlushingHog } from 'lib/components/hedgehogs'
+import useResizeObserver from 'use-resize-observer'
+
+import { BillingProductV2Type } from '~/types'
+
+import { billingLogic } from './billingLogic'
+import { billingProductLogic } from './billingProductLogic'
+import { PlanComparisonModal } from './PlanComparison'
+
+export const BillingCTAHero = ({ product }: { product: BillingProductV2Type }): JSX.Element => {
+    const { width, ref: billingHeroRef } = useResizeObserver()
+
+    const { redirectPath } = useValues(billingLogic)
+    const { isPlanComparisonModalOpen, billingProductLoading } = useValues(billingProductLogic({ product }))
+    const { toggleIsPlanComparisonModalOpen, setBillingProductLoading } = useActions(billingProductLogic({ product }))
+
+    // TODO(@zach): add multiple variations of this copy
+    return (
+        <div className="flex relative justify-between items-center rounded-lg bg-mark" ref={billingHeroRef}>
+            <div className="p-4">
+                {/* <h1>Add your credit card to unlock all the features.</h1> */}
+                <h1>Upgrade to unlock all the features.</h1>
+                <h1 className="text-danger">Only pay for what you use.</h1>
+                <p className="mt-2 mb-0">
+                    You're currently on the free plan. It's free but limited in features. Add your credit card to
+                    upgrade to our paid plan where you pay per use (after the generous free tier).
+                </p>
+                <div className="flex justify-start space-x-2">
+                    <LemonButton
+                        className="mt-4 inline-block"
+                        onClick={() => toggleIsPlanComparisonModalOpen()}
+                        type="secondary"
```

Comment:
This button looked odd:

<img width="488" alt="Arc 2024-06-17 18 57 36" src="https://github.com/PostHog/posthog/assets/18598166/2336edf4-45bd-4b0d-8112-d5d32731cf1a">

I've pushed a change to swap the order of the buttons (upgrade first, then compare) and change them to look like this:

<img width="560" alt="Arc 2024-06-17 19 02 12" src="https://github.com/PostHog/posthog/assets/18598166/0d5dfca6-fe17-474d-870f-9bc60c40eb9e">

Reply:
Thanks :)